var/spack/repos/builtin/packages/py-pytables/package.py

```diff
@@ -12,13 +12,20 @@ class PyPytables(PythonPackage):
     homepage = "http://www.pytables.org/"
     url      = "https://github.com/PyTables/PyTables/archive/v3.3.0.tar.gz"
 
+    version('3.5.1', sha256='fda2e5071ce064ae1e808623e4c6166e6ab5f0f6152043991c5151d4e2622f2e')
     version('3.4.4', '2cd52095ebb097f5bf58fa65dc6574bb')
     version('3.3.0', '056c161ae0fd2d6e585b766adacf3b0b')
     version('3.2.2', '7cbb0972e4d6580f629996a5bed92441',
             url='https://github.com/PyTables/PyTables/archive/v.3.2.2.tar.gz')
 
+    variant('blosc', default=True, description='Use spack provided c-blosc.')
+    variant('lzo', default=False, description='Use optional lzo')
+    variant('bzip2', default=False, description='Use optional bzip2')
```

Comment:
Regarding these variants:

* Based on the comment for `blosc`, it sounds like this package always needs `blosc` but can be directed to use the Spack version. When that option is available, it's generally preferable to always build with the Spack instance (that is to say if I understand the description, this shouldn't be a variant).
* `bzip2` will automatically be pulled in as a dependency, but as a general rule whenever your package directly depends on another package, it's worth stating that explicitly (I don't see a `depends_on(bzip2)` below)